src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/products_v2/product.yml

```
@@ -34,6 +34,14 @@ admin_products_preview:
   requirements:
     productId: \d+
 
+admin_products_export:
+  path: /export
+  methods: [ GET ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Catalog\Product\ProductController::exportAction
+    _legacy_controller: AdminProducts
+    _legacy_link: AdminProducts:exportproduct
```

The `exportproduct` doesn't seems to be in the legacy `controllers/admin/AdminProductsController.php`.
Maybe, we can simply do this:

```suggestion
    _legacy_link: AdminProducts
```

---

@boherm yeah, it's will be better. Commited